File: INFA/zadanie/zadanie.cpp
#include <iostream>
#include <cmath>
using namespace std;
int dziesietny, x = 0, y, wypisywanie;

int main()
{
	cout << "Podaj liczbe w sysytemie dziesietnym: ";
	cin >> dziesietny;

	wypisywanie = dziesietny;
	y = dziesietny;
	while (y > 0)
	{
		y = y / 2;
		x++;
	}

	int T[x];
	x = -1;

	while (dziesietny > 0)
	{
		x++;
		T[x] = dziesietny % 2;
		dziesietny = dziesietny / 2;
	}

	cout << "(" << wypisywanie << ")"
		 << " W systemie binarnym to: ";

	while (x >= 0)
	{
		cout << T[x];
		x--;
	}

	return 0;
}

File: INFA/zadanie/zadanie.py
a = int()
b = int()
x = int(input("Podaj liczbe w systemie dwojkowych: "))

xxx = x

while x > 0:
    b += int(x % 10 * pow(2, a))
    a = int(a + 1)
    x = int(x / 10)
print(xxx, " W systemie dziesietnym to: ", b)
